inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -348,7 +372,7 @@ private function update_used_css( int $id, array $data ) {
 	 *
 	 * @return string HTML content.
 	 */
-	private function remove_used_css_from_html( string $html, array $unprocessed_css ) : string {
+	public function remove_used_css_from_html( string $html, array $unprocessed_css ): string {
```

Comment:
Did you intend to make this public again. I'm not seeing anywhere outside of this class that uses it?

Reply:
Not intended, I think it happened because of the merge conflict with the spacing fix for the typehint